### ISSUE TITLE
6.x changed/broke something related to DefaultRowHeight, not sure what

### ISSUE THREAD

**Issue by ses4j** [EPPlusSoftware/EPPlus#727]
Here's a reproduction of a weird row height issue I started seeing in 6.x.  5.x worked as expected, this only cropped up when I migrated to 6.x.  For some reason, the "merge.xlsx" in the zipfile seems to be read in a weird way.  EPPlus decides the DefaultRowHeight and the non-explicit rows in merge.xlsx are all "90" high.  I don't know why, and Excel doesn't show them that way.

![image](https://user-images.githubusercontent.com/435529/196059630-bf89bd9b-beae-4d58-8c00-1d0653a92d8d.png)

Output with `<PackageReference Include="EPPlus" Version="5.8.14" />`, which seems fine:
```
BEFORE  IN   Locations: defaultrowheight=15 False, row heights=15, 15, 15, 15
DURING  MRG  Cat terms: defaultrowheight=12.75 False, row heights=12.75, 12.75, 20.1, 12.75
DURING  OUT  Cat terms: defaultrowheight=15 False, row heights=12.75, 12.75, 20.1, 15
AFTER  OUT  Locations: defaultrowheight=15 False, row heights=15, 15, 15, 15
AFTER  OUT  Cat terms: defaultrowheight=15 False, row heights=12.75, 12.75, 20.1, 15
```

Output with `<PackageReference Include="EPPlus" Version="6.0.8" />`, which seems incorrect:
```
(.venv) C:\temp\fun\rowheightissue>dotnet C:\temp\fun\rowheightissue\bin\Debug\net6.0\rowheightissue.dll
BEFORE  IN   Locations: defaultrowheight=15 False, row heights=15, 15, 15, 15
DURING  MRG  Cat terms: defaultrowheight=90 False, row heights=90, 90, 20.1, 90
DURING  OUT  Cat terms: defaultrowheight=15 False, row heights=90, 90, 20.1, 15
AFTER  OUT  Locations: defaultrowheight=15 False, row heights=15, 15, 15, 15
AFTER  OUT  Cat terms: defaultrowheight=15 False, row heights=90, 90, 20.1, 15
```

**Comment by JanKallman** [comment]
I'll have a look at this.

**Comment by JanKallman** [comment]
I can't see the zip-file here. Can you attach it?

**Comment by ses4j** [comment]
Ugh, sorry.  Here you go.

[rowheightissue.zip](https://github.com/EPPlusSoftware/EPPlus/files/9802250/rowheightissue.zip)

**Comment by JanKallman** [comment]
I have added a fix for this issue. The problem was that the Height was copied at all times, also when the row had the default height. As the workbooks has different Normal style fonts, the row height differed. You can try it using our develop feed ... https://github.com/EPPlusSoftware/EPPlus/wiki/Using-our-Develop-Nuget-Feed

**Comment by JanKallman** [comment]
Fixed in EPPlus 6.1
